### PR TITLE
libmicroutils: New package.

### PR DESCRIPTION
**Pull request by silvioprog**
Hello masters,

This PR adds a new cross-platform C library called `libmicroutils` for handling strings, files, regex, hashs, among others.

Home page: https://silvioprog.github.io/microutils
Github repo: https://github.com/silvioprog/microutils
HTML doc: https://silvioprog.github.io/microutils/doc/html
PDF doc: https://silvioprog.github.io/microutils/doc/pdf
Travis CI tests: https://travis-ci.org/silvioprog/microutils
Launchpad project: https://launchpad.net/microutils

Thank you!

### CODE REVIEW

**Comment by silvioprog** [comment]
Hello,

Any change to add this library into mingw-packages?

Thank you!

File: mingw-w64-libmicroutils/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Maintainer: Silvio Clecio <silvioprog@gmail.com>
+
+_realname=libmicroutils
+pkgbase=mingw-w64-${_realname}
+pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
+pkgver=4.0.0
+pkgrel=1
+pkgdesc="Micro cross-platform C library for handling strings, files, regex, hashs and among others. (${MINGW_PACKAGE_PREFIX})"
```
Comment (Alexpux) [inline mingw-w64-libmicroutils/PKGBUILD]: Remove (${MINGW_PACKAGE_PREFIX}) from pkgdesc

File: mingw-w64-libmicroutils/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+  make
+}
+
+package() {
+  cd "${srcdir}/build-${MINGW_CHOST}"
+  make DESTDIR="${pkgdir}" install
+
+  install -Dm644 ${srcdir}/${_realname}-${pkgver}/COPYING ${pkgdir}${MINGW_PREFIX}/share/licenses/${_realname}/LICENSE
+}
```
Comment (Alexpux) [inline mingw-w64-libmicroutils/PKGBUILD]: Add new line at the end of file

File: mingw-w64-libmicroutils/PKGBUILD (hunk 1 above)
Comment (silvioprog) [inline mingw-w64-libmicroutils/PKGBUILD]: Done! :-)

File: mingw-w64-libmicroutils/PKGBUILD (hunk 2 above)
Comment (silvioprog) [inline mingw-w64-libmicroutils/PKGBUILD]: Done! :-)

**Comment by silvioprog** [comment]
The library is now in a new release: v4.2.0. :-)

**Comment by Alexpux** [comment]
Rebase changes on top of current branch

**Comment by silvioprog** [comment]
Done. Please let me know if it works. :-)

**Pull request merged**